mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -355,8 +354,15 @@ object Tool extends LazyLogging {
             logger.error(text)
 
           case FailureMessage(text) =>
-            Console.err.println("Please report an issue at: " + ISSUES_LINK, e)
             logger.error(text, e)
+            val absPath = ReportGenerator.prepareReportFile(
+                specFile,
+                s"${Version.version} build ${Version.build}"
+            )
+            Console.err.println(
+                s"Please report an issue at $ISSUES_LINK: $e\nA bug report template has been generated at [$absPath]."
```

Comment:
```suggestion
                s"Please report an issue at $ISSUES_LINK: $e\nA bug report has been generated at [$absPath]."
```

Iiuc, we are generating the report using the template?

Reply:
Technically, the user is still meant to fill in "Expected Behavior". I'll add a prompt to do that in the stdout string.